pkg/epinio/detail/applications.vue

```diff
@@ -5,20 +5,20 @@ import SimpleBox from '@shell/components/SimpleBox.vue';
 import ConsumptionGauge from '@shell/components/ConsumptionGauge.vue';
 import { EPINIO_PRODUCT_NAME, EPINIO_TYPES } from '../types';
 import ResourceTable from '@shell/components/ResourceTable.vue';
-import Tabbed from '@shell/components/Tabbed/index.vue';
-import Tab from '@shell/components/Tabbed/Tab.vue';
 import PlusMinus from '@shell/components/form/PlusMinus.vue';
 import { epinioExceptionToErrorsArray } from '../utils/errors';
-
+import ApplicationCard from '~/shell/components/cards/ApplicationCard.vue';
```

Comment:
There's a frustrating issue that VSCode will default to imports with `~` when we don't want them to. For instance this should be `@shell`